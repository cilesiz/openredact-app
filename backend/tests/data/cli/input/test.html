<!DOCTYPE html>

<html lang="de">
  <head>
    <title>Test</title>
    <meta charset="utf-8" />
  </head>
  <body class="no-js foo">
    <div>Meine E-Mail Adresse lautet test@mail.de.</div>
    <div>Sie ist in Deutschland registriert.</div>
  </body>
</html>
