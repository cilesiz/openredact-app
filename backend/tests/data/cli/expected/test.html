<!DOCTYPE html>

<html lang="de">
  <head>
    <title>Test</title>
    <meta charset="utf-8" />
  </head>
  <body class="no-js foo">
    <div>Meine E-Mail Adresse lautet Email 1.</div>
    <div>Sie ist in XXXXXXXXXXX registriert.</div>
  </body>
</html>
